cvat/apps/engine/views.py

```diff
@@ -928,11 +1128,58 @@ def upload_finished(self, request):
                         status=status.HTTP_400_BAD_REQUEST)
 
     @extend_schema(methods=['POST'],
-        summary='Method permanently attaches images or video to a task. Supports tus uploads, see more https://tus.io/',
+        summary="Method permanently attaches data (images, video etc.) to a task",
+        description=textwrap.dedent("""\
+            Allows to upload data to a task.
+            Supports the TUS open file uploading protocol (https://tus.io/).
+
+            Implements the following protocols:
+            a. A single Data request
+            b.1. An Upload-Start request
+            b.2.a. The regular TUS protocol requests (Upload-Length + Chunks)
+            b.2.b. Upload-Multiple requests
+            b.3. An Upload-Finished request
+
+            Requests:
+            - Data - POST, no extra headers or 'Upload-Start' + 'Upload-Finish' headers.
+              Contains Data in the body.
+            - Upload-Start - POST, has an 'Upload-Start' header.
+              Can contain upload metainfo in the body.
+            - Upload-Length - HEAD, has an 'Upload-Length' header (read the TUS protocol)
+            - Chunk - PATCH (read the TUS protocol).
+            - Upload-Finish - POST, has an 'Upload-Finish' header. Can contain Data in the body.
+            - Upload-Multiple - POST, has a 'Upload-Multiple' header. Contains Data in the body.
+
+            The 'Upload-Start' request allows to specify file order for TUS requests.
```

Comment:
Wouldn't it also work for non-TUS (Upload-Multiple) requests?

Reply:
That's better, but now the question is, why is the field called `tus_file_order` and not just `file_order`?

Reply:
The idea was to relate to the whole "CVAT-TUS" protocol, but it may be confused. Probably, it can be renamed to `upload_file_order`, because `file_order` can also be confusing - it can mean it is always applied, while it's not supposed for this.